### ISSUE TITLE
Left double click to open message in conversation 

### ISSUE THREAD

**Issue by dbogdanov** [protz/thunderbird-conversations#1408]
* Thunderbird version: 68.3.1
* Thunderbird Conversation version: 3.0.3
* Operating System: macOS High Sierra 10.13.6

Left double-clicking on a message won't open it in a new tab since installing Thunderbird Conversation add-on. Is this configurable?

**Comment by Standard8** [comment]
Duplicate of #1407.

Not configurable - it is just broken at the moment. I've already fixed it, we're just waiting on the add-on reviewers to approve it.

The workaround is to click on the left-hand symbol at the end of the summary bar - <img width="43" alt="Screenshot 2020-01-06 at 10 28 56" src="https://user-images.githubusercontent.com/520844/71812576-6c052100-306f-11ea-818c-c65c740c6443.png">

**Comment by Standard8** [comment]
Duplicate of #1407

**Comment by Standard8** [comment]
(Sorry for the extra noise, I was figuring out about github's duplicate annotations).

**Comment by Standard8** [comment]
Good news, this was approved an hour ago, so it should be available via Three-bar-menu -> Add-ons -> Add-ons, then click on Extensions, followed by the cog symbol and check for updates.

You may need to restart Thunderbird for it to full take effect.
